dspace/config/spiders/agents/COUNTER_Robots_list.txt

```diff
@@ -123,12 +125,12 @@ ichiro
 iktomi
 ilse
 Indy Library
-^integrity\/\d
+^integrity\/[0-9]
 internetseer
 intute
 iSiloX
 iskanie
-^java\/\d{1,2}.\d
```

Comment:
Cambia la j por J
se pierde la cardinalidad  {1,2}
Debería ser 
^java\/[0-9]{1,2}\.[0-9]

Reply:
Con esto pasa algo interesante. Desde COUNTER [recomiendan](https://github.com/atmire/COUNTER-Robots#case-insensitivity-of-pattern-matching) que se haga un matcheo *case_insensitive*, el problema es que en Solr para los campos "String" no se puede indicar esto. Cuando busqué esta expresión en Solr, todas los matcheos eran por "Java..." en vez de "java".
En el caso de la cardinalidad no lo habia entendido bien.